tests/unit/system/classes/UpdatesControllerTest.php

```diff
@@ -14,20 +14,12 @@ public function testGetPluginVersionFile()
         $controller = $this->getMockBuilder(Updates::class)->disableOriginalConstructor()->getMock();
 
         $expectedVersions = [
-            '1.3.2' => [
-                'Added support for Translate plugin. Added some new languages.',
-            ],
-            '1.3.1' => [
-                'Minor bug fix Please see changelog',
-            ],
-            '1.3.0' => [
-                '!!! We\'ve refactored major parts of this plugin. Please see the website for more information.',
-            ],
```

Comment:
Why were these test cases removed?

Reply:
Same reason with the as above

Reply:
@jaxwilko is correct - that test was part of the Yaml upgrade which we didn't proceed with.